src/model/clean.ts

```diff
@@ -11,18 +12,14 @@ export const bigBullets = (s: string) =>
 const { window } = new JSDOM('');
 const DOMPurify = createDOMPurify(window);
 
-export const clean = compose(
-	bigBullets,
-	(s: string) =>
-		minify(s, {
-			collapseWhitespace: true,
-			minifyCSS: true,
-			minifyJS: true,
-		}),
-	// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-	DOMPurify.sanitize,
+export const clean = compose(bigBullets, (s: string) =>
```

Comment:
We could rename this to something like `minimise` now that it's purpose is a bit more narrow/just minification?

Reply:
Good point - shall do.